src/detext/resources/vocab-100.txt

```diff
@@ -0,0 +1,100 @@
+[PAD]
```

Comment:
Why do we need this file?

Reply:
The other vocab file is a bit too small. For local testing, I used this vocab with 100 tokens to run on the sample data.

Reply:
removed this one in the latest update.